src/components/home/home.component.jsx

```diff
@@ -206,15 +207,15 @@ const Home = () => {
             <div className={classes.col2}>
               <h3 className={classes.blogTitle}>
                 <a
-                  href='https://blog.hermez.io/hermez-bug-bounty-program/'
+                  href={blogPosts[0].url}
```

Comment:
Couldn't all this happen in a for loop so you only repeat the blog elements once?

Reply:
yeah right! updated